tests/arch/x86/hybrid/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2023 Intel Corp.
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(intel_hybrid)
```

Comment:
As mentioned elsewhere, I'd really prefer to see "Intel hybrid CPU support" be a separate PR instead of mixed with arch-level changes.